subprojects/docs/src/docs/userguide/javaTesting.adoc

```diff
@@ -15,8 +15,55 @@
 [[java_testing]]
 == Testing in Java & JVM projects
 
+////
+To run integration tests:
+
+   ./gradlew :integTest:intTestImage :docs:extractSamples :integTest:integTest --tests "*SamplesJavaTestingIntegrationTest"
+////
+
 Testing on the JVM is a rich subject matter. There are many different testing libraries and frameworks, as well as many different types of test. All need to be part of the build, whether they are executed frequently or infrequently. This chapter is dedicated to explaining how Gradle handles differing requirements between and within builds, with significant coverage of how it integrates with the two most common testing frameworks: https://junit.org/[JUnit] and https://testng.org/[TestNG].
 
+It explains:
+
+ * Ways to control how the tests are run (<<sec:test_execution,Test execution>>)
+ * How to select specific tests to run (<<sec:test_filtering,Test filtering>>)
+ * What test reports are generated and how to influence the process (<<test_reporting,Test reporting>>)
+ * How Gradle finds tests to run (<<sec:test_detection,Test detection>>)
+ * How to make use of the major frameworks' mechanisms for grouping tests together (<<test_grouping,Test grouping>>)
+
+But first, we look at the basics of JVM testing in Gradle.
+
+[[sec:java_testing_basics]]
+=== The basics
```

Comment:
❓ Would "Testing configuration basics" be usefully specific here?

Reply:
Not that specific wording, no. It's easy to read as "Testing the basics of configuration". It's also not specific to the configuration of the tests. It's really a simple overview of what the Java Plugin does for you as well as an introduction to the relevant tasks. Would you prefer "Overview"? Or "An introduction to Java testing"? Or "The basics/An overview of Java testing"? It's the first section of the chapter, and it's clear what the chapter is about, so I'm not sure what the value of being specific is here. But I don't feel particularly strongly either way.